docs/library/graphing/funnelchart.md

```diff
@@ -4,13 +4,18 @@ components:
     - rx.recharts.Funnel
 ---
 
-# Funnel Chart
-
 ```python exec
 import reflex as rx
-from pcweb.templates.docpage import docdemo, docgraphing
 import random
+```
+
+# Funnel Chart
+
+A funnel chart is a graphical representation used to visualize how data moves through a process. In a funnel chart, the dependent variable’s value diminishes in the subsequent stages of the process. It can be used to demonstrate the flow of users through for example a business or sales process.
```

Comment:
```suggestion
A funnel chart is a graphical representation used to visualize how data moves through a process. In a funnel chart, the dependent variable’s value diminishes in the subsequent stages of the process. It can be used to demonstrate the flow of users through a business or sales process.
```